File: 8kyu/total-amount-of-points/total-amount-of-points.py
def points(games):
  arr = list(map(lambda game: game.split(":"), games))
  result = 0
  for points in arr:
    if points[0] > points[1]:
      result += 3
    if points[0] == points[1]:
      result += 1
  return result


print(points(['1:0','2:0','3:0','4:0','2:1','3:1','4:1','3:2','4:2','4:3']))

File: 8kyu/total-amount-of-points/total-amount-of-points.ts
export function points(games : string[]): number {
  const arr = games.map(game => game.split(":").map(Number));

  return arr.reduce((acc, [a, b]) => {
    if (a > b) acc += 3
    if (a === b) acc += 1

    return acc
  }, 0)
}